src/index.spec.ts

```diff
@@ -110,12 +107,17 @@ describe('EppoJSClient E2E test', () => {
   };
 
   beforeAll(async () => {
-    mock.setup();
+    global.fetch = jest.fn(() => {
+      const ufc = readMockUfcResponse(MOCK_UFC_RESPONSE_FILE);
+
+      return Promise.resolve({
+        ok: true,
+        status: 200,
+        json: () => Promise.resolve(ufc),
+      });
+    }) as jest.Mock;
```

Comment:
I would argue that these tests are redundant as they are handled now within the commons library.

Reply:
Yeah, I need to think about the value of the tests.